packages/sdk-bundles-api/src/replay-orchestrator/sdk-to-bundle/execute-replay.ts

```diff
@@ -184,7 +185,7 @@ export interface ReplayExecutionOptions {
   /**
    * If populated, each header will be injected into all requests when fetching resources during a replay.
    */
-  customRequestHeaders?: Record<string, string>;
+  customRequestHeaders?: InjectableRequestHeader[];
```

Comment:
The readme mentions that changing a type is unsafe but since this has never been used I think it's fine?